alert.go

```diff
@@ -780,8 +792,10 @@ func (a *AlertNode) renderMessageAndDetails(id, name string, t time.Time, group
 
 func (a *AlertNode) handlePost(post *pipeline.PostHandler, ad *AlertData) {
 	bodyBuffer := a.bufPool.Get().(*bytes.Buffer)
-	defer a.bufPool.Put(bodyBuffer)
-	bodyBuffer.Reset()
+	defer func() {
+		bodyBuffer.Reset()
+		a.bufPool.Put(bodyBuffer)
```

Comment:
I don't think there is some of concern, since sync.Pool isn't smart enough with the buffer pools, 
If we would other buffer pool that will have large buffer and will give us segments per request, it will be smarter to create buffer pool for "large objects" (like handlePost, json helper) and "small objects" id template.